trello_items.py

```diff
@@ -56,40 +97,63 @@ def create_item(title, description):
     """
     url = "https://api.trello.com/1/cards"
     query = {
-        'key': key,
-        'token': token,
+        'key': os.getenv('API_KEY'),
+        'token': os.getenv('API_TOKEN'),
         'name': title, 
         'desc': description,
-        'pos': len(get_items()) + 1, 
-        'idList': todo_list_id # this is the id for the 'To Do' list
+        'idList': os.getenv('TODO_LIST_ID') # this is the id for the 'To Do' list
+    }
+    response = requests.request("POST", url, params=query)
+    return response
+
+
+def start_item(item_id):
+    """
+    Moves a card to the 'DOING' list of the board.
+    """
+    url = "https://api.trello.com/1/cards/{}".format(item_id)
+    query = {
+        'key': os.getenv('API_KEY'),
+        'token': os.getenv('API_TOKEN'),
+        'idList': os.getenv('DOING_LIST_ID') # this is the id for the 'Doing' list
     }
-    card = requests.request("POST", url, params=query)
-    return card
+    return requests.request("PUT", url, params=query)
 
 
 def complete_item(item_id):
     """
     Moves a card to the 'DONE' list of the board.
     """
-    id_list = done_list_id # this is the id for the 'Done' list
     url = "https://api.trello.com/1/cards/{}".format(item_id)
     query = {
-        'key': key,
-        'token': token,
-        'idList': id_list
+        'key': os.getenv('API_KEY'),
+        'token': os.getenv('API_TOKEN'),
+        'idList': os.getenv('DONE_LIST_ID') # this is the id for the 'Done' list
```

Comment:
I don't think the comment really explains any more than the variable name "DONE_LIST_ID" manages.